.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: required
+dist: trusty
+language: cpp
+compiler:
+- gcc
```

Comment:
Re: [clang](https://docs.travis-ci.com/user/languages/cpp) and [multi-os](https://docs.travis-ci.com/user/multi-os/) yes, we can - and I think we should (like I said, this is a baseline config).